src/renderer/wg_engine/tvgWgBindGroups.h

```diff
@@ -23,7 +23,7 @@
 #ifndef _TVG_WG_BIND_GROUPS_H_
 #define _TVG_WG_BIND_GROUPS_H_
 
-#include "tvgWgCommon.h"
```

Comment:
how about putting the header `#include <webgpu/webgpu.h>` in `tvgWgCommon.h` ?

Reply:
@hermet 
_tvgWgBindGroups.h_ also reguired _#include <webgpu/webgpu.h>_
But you still have an access to all webgpu instances if you will include _tvgWgCommon.h_ somewhere 

As _alternalive_, we can get rid tvgWgBindGroups.h and move BindGroup declaration into _tvgWgCommon.h_
its also make a sence

Reply:
> @SergeyLebedkin I mean you can include `tvgWgCommon.h`, which could be a common and general header for wg implementation if it include the `webgpu.h`

tvgWgCommon.h use tvgWgBindGroups.h, because tvgWgCommon containe Context, that use BindGroups instance. So context must know about BindGroups, so they must be declared before. We can use forward declatation, but I want to use instalnce access but not pointer to the layouts in context

Reply:
@SergeyLebedkin You can merge them as `tvgWgCommon.h`

Reply:
> @SergeyLebedkin You can merge them as `tvgWgCommon.h`

Yes, its also make a sence...

Reply:
@SergeyLebedkin And it's optional, you can submit patch.